smart_card_connector_app/src/window-logs-exporting.js

```diff
@@ -66,8 +69,8 @@ function exportLogsClickListener(e) {
   if (!isExportLogsAvailable)
     return;
 
-  goog.dom.classlist.add(
-      exportLogsElement, EXPORT_LOGS_ELEMENT_EXPORTING_CLASS);
+  exportLogsElement.innerText =
```

Comment:
"innerText" is a non-standard feature, basically - IE legacy.
Better to use "textContent" or goog.dom.setTextContent.